snippets/server-platforms/servicestack/setup.md

```diff
@@ -1,12 +1,13 @@
 ```cs
-private void ConfigureAuth(Funq.Container container)
-{
+
```

Comment:
@iannyanes Can you please fix the indentation on this snippet. It looks pretty bad.

![image](https://cloud.githubusercontent.com/assets/11715799/18242871/17143254-7361-11e6-91e9-96c2c3501124.png)